### PR TITLE
docs: marketplace page

### PR DESCRIPTION
**Pull request by ivikash**
Marketing Page Updates
- Amazon Q: - Demo - https://github.com/aws/aws-toolkit-vscode/blob/8f0b4988259cb73a3f15188c984cf4ef53b65b47/packages/amazonq/README.md
  - Added badges
  - Moved Getting Started to top of the content
  - Created a Section called Features and added all the contents beneath that
  - Update gif links to `https://raw.githubusercontent.com` to skip CORS and render images.

- AWS Toolkit: Demo - https://github.com/aws/aws-toolkit-vscode/blob/8f0b4988259cb73a3f15188c984cf4ef53b65b47/packages/toolkit/README.md
  - Moved Getting Started to top of the content
  - Update gif links to `https://raw.githubusercontent.com` to skip CORS and render images.

## License

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Comment by ankityyz** [comment]
Looks great!

File: packages/amazonq/README.md [hunk 1]
```diff
@@ -1,57 +1,62 @@
 ## Inline code suggestions
 
+[![Marketplace Version](https://img.shields.io/vscode-marketplace/v/AmazonWebServices.amazon-q-vscode.svg)
```
Comment (justinmk3) [inline packages/amazonq/README.md:3]: this is redundant with our changelog and "Release" link in github

File: packages/amazonq/README.md [hunk 2]
```diff
@@ -1,57 +1,62 @@
...
 
-## Security scans
+### Security scans
 
 Analyze and fix security vulnerabilities in your project.
 
-![Fix security vulnerability demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/security-scan.gif)
+![Fix security vulnerability demo](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/security-scan.gif)
 
 <sub>[_10 languages supported including Python, TypeScript, C#, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/security-scans.html)</sub>
```
Comment (justinmk3) [inline packages/amazonq/README.md:34]: why is this forced to be so small? it doesn't look great

image

File: packages/amazonq/README.md [hunk 3]
```diff
@@ -1,57 +1,62 @@
 ## Inline code suggestions
 
+[![Marketplace Version](https://img.shields.io/vscode-marketplace/v/AmazonWebServices.amazon-q-vscode.svg)
+![Marketplace Downloads](https://img.shields.io/vscode-marketplace/d/AmazonWebServices.amazon-q-vscode.svg)](https://marketplace.visualstudio.com/items?itemName=AmazonWebServices.amazon-q-vscode)
+
 Code faster with inline code suggestions as you type.
 
-![Inline code suggestion demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/inline.gif)
+## Getting Started
```
Comment (justinmk3) [inline packages/amazonq/README.md:8]: this should be h1

File: packages/amazonq/README.md [hunk 4]
```diff
@@ -1,57 +1,62 @@
...
+
+**Free Tier** - create or log in with an AWS Builder ID (no AWS account needed!).
+
+**Pro Tier** - if your organization is on the Amazon Q Developer Pro tier, log in with single sign-on.
+
+![Authentication gif](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/auth-Q.gif)
+
+## Features
```
Comment (justinmk3) [inline packages/amazonq/README.md:16]: h1
```suggestion
# Features
```

File: packages/amazonq/README.md [hunk 5]
```diff
@@ -1,57 +1,62 @@
...
+
+![Inline code suggestion demo](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/inline.gif)
 
 <sub>[_15+ languages supported including Python, TypeScript, Rust, Terraform, AWS Cloudformation, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-language-ide-support.html)</sub>
 
-## Chat
+### Chat
```
Comment (justinmk3) [inline packages/amazonq/README.md:22]: h2

over-using the "nesting" of headings leads to unnecessarily small headings. h1 can be used for the "main" headings.

File: packages/amazonq/README.md [hunk 6]
```diff
@@ -1,57 +1,62 @@
...
 
-## Agent for software development
+### Agent for software development
 
 Amazon Q can plan and implement new functionality across multiple files in your workspace.
 
 <sub>Type “/” in chat to open the quick actions menu and choose the “/dev” action.</sub>
 
-![Agent for software development demo](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/dev.gif)
+![Agent for software development demo](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/dev.gif)
```
Comment (justinmk3) [inline packages/amazonq/README.md:42]: does it work if you use `raw/HEAD` ?

that is what vsce [does internally](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#vscode-marketplace)

File: packages/toolkit/README.md [hunk 7]
```diff
@@ -1,5 +1,12 @@
 The AWS Toolkit lets you interact with AWS directly from VS Code. Ready to install? See [Getting Started](#getting-started).
 
+## Getting Started
+
+1. Open the AWS Toolkit extension
+2. Sign in with your AWS credentials
+
+![getting-started](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/auth-Toolkit.gif)
```
Comment (justinmk3) [inline packages/toolkit/README.md:8]: good idea

File: packages/amazonq/README.md [hunk 8]
```diff
@@ -1,57 +1,62 @@
...
 
 <sub>[_15+ languages supported including Python, TypeScript, Rust, Terraform, AWS Cloudformation, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-language-ide-support.html)</sub>
 
-## Chat
+### Chat
 
 Generate code, explain code, and get answers to questions about software development.
 
-![Generate code using chat](https://github.com/aws/aws-toolkit-vscode/raw/HEAD/docs/marketplace/vscode/amazonq/chat.gif)
+![Generate code using chat](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/master/docs/marketplace/vscode/amazonq/chat.gif)
```
Comment (justinmk3) [inline packages/amazonq/README.md:26]: does it work if relative paths are used instead? see https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#vscode-marketplace

File: packages/amazonq/README.md (hunk 6 above)
Comment (ivikash) [reply packages/amazonq/README.md:42]: Yes, it does. Updated in the next revision.

File: packages/amazonq/README.md [hunk 9]
```diff
@@ -1,22 +1,37 @@
...
+
+**Free Tier** - create or log in with an AWS Builder ID (no AWS account needed!).
+
+**Pro Tier** - if your organization is on the Amazon Q Developer Pro tier, log in with single sign-on.
+
+![Authentication gif](https://raw.githubusercontent.com/aws/aws-toolkit-vscode/HEAD/docs/marketplace/vscode/amazonq/auth-Q.gif)
+
+# Features
 
-<sub>[_15+ languages supported including Python, TypeScript, Rust, Terraform, AWS Cloudformation, and more_](https://docs.aws.amazon.com/amazonq/latest/qdeveloper-ug/q-language-ide-support.html)</sub>
```
Comment (justinmk3) [inline packages/amazonq/README.md:7]: `<sub>` is still in other parts of this doc, can we remove it there too. since we are changing stuff anyway.

Comment (ivikash) [reply packages/amazonq/README.md:7]: Done.

**Pull request merged**